cloudinit/net/openbsd.py

```diff
@@ -43,7 +43,7 @@ def start_services(self, run=False):
                     ["dhcpleasectl", "-w", "30", interface], capture=True
                 )
         else:
-            net.dhcp.IscDhclient.kill_dhcp_client()
+            subp.subp(["pkill", "dhclient"], capture=True, rcs=[0, 1])
```

Comment:
Hi, I'm not sure this needed. `kill_dhcp_client()` does the same thing already.

Reply:
Hi and sorry, you're right. I don't know what's happened here, I didn't wanted to change this. It's the line as it is in my clone from main at that day, so just ignore that please.